draft-ietf-httpapi-ratelimit-headers.md

```diff
@@ -320,30 +320,34 @@ and can be sent in a trailer section.
 
 ## RateLimit-Remaining {#ratelimit-remaining-header}
 
-The `RateLimit-Remaining` response field indicates the remaining `quota-units` defined in {{request-quota}}
-associated to the client.
+The `RateLimit-Remaining` response field indicates the remaining `quota-units`
+defined in {{request-quota}} associated to the client.
 
-The header value is
+`RateLimit-Remaining` is a Item Structured Field
+{{!I-D.ietf-httpbis-header-structure}}. Its value MUST be an Integer (Section
+3.3.1 of {{I-D.ietf-httpbis-header-structure}}). Its ABNF is:
 
 ~~~
-   RateLimit-Remaining = quota-units
+   RateLimit-Remaining = sf-integer
 ~~~
 
-This header MUST NOT occur multiple times
-and can be sent in a trailer section.
+The value of `RateLimit-Remaining` MUST be non-negative.
 
-Clients MUST NOT assume that a positive `RateLimit-Remaining` value is
-a guarantee of being served.
-
-A low `RateLimit-Remaining` value is like a yellow traffic-light: the red light
-may arrive suddenly.
+`RateLimit-Remaining` MUST NOT occur multiple times. It MAY be sent in a trailer
+section.
 
-One example of `RateLimit-Remaining` use is below.
```

Comment:
While I find your form more readable, other RFCs use this formula "X example[s] of ... is below". This is possibily related to the fact that in case of multiple lines, the reader might not know beforehand whether we are providing one or more examples.

Have a look at https://httpwg.org/http-core/draft-ietf-httpbis-semantics-latest.html#rfc.section.5.6.7.p.3 . Probably we could `s,One,An`

Reply:
In this spirit of your first suggestion, maybe we can do this in an editorial pass later? That way I can make the diff as small as possible.